tools/azure-sdk-tools/devtools_testutils/mgmt_testcase.py

```diff
@@ -76,6 +80,7 @@ def __init__(
             replay_patches=replay_patches,
             **kwargs
         )
+        self.recording_processors.append(self.re_replacer)
```

Comment:
Do you need access to the `RENameReplacer` specifically? I think you can replace this with: `self.recording_processors.append(RENameReplacer())` and remove the `self.re_replacer` line.

Reply:
Because I need to add regex to this replacer.